modules/component-store/src/component-store.ts

```diff
@@ -92,7 +93,10 @@ export class ComponentStore<T extends object> implements OnDestroy {
         .pipe(
           concatMap((value) =>
             this.isInitialized
-              ? of(value).pipe(withLatestFrom(this.stateSubject$))
+              ? // Push the value into queueScheduler
+                of(value, queueScheduler).pipe(
```

Comment:
Not sure if we should worry about it, but passing a scheduler to `of` is [deprecated](https://github.com/ReactiveX/rxjs/blob/master/src/internal/observable/of.ts) and will be removed in v8.

Reply:
Thanks Tim. Didn't know about that.
As of right now - `scheduled` isn't typed correctly for single values. It's focused more on Array, promises and observables, so I had to wrap it with array.